1_Mechanics/02_GMGuide.md

```diff
@@ -89,54 +270,425 @@ the probability by as much as 25% for a DR 3 check.
-
-### Contested checks
-
-Examples
-1. Tackle target: Attacker Finesse vs. Target Athletics
-2. Convince Target, truthfully: Bluffing vs. Detection
-3. Convince Target, deceptively: Bluffing vs. Detection
-
-## Running an Adventure
-
-`MISSING`
-
-## Managing Players (Expectations & Realities)
-
-`MISSING`
-
-## Crunch
-
-`MISSING`
-
-## Now What?
-
-`MISSING`
+- VIT Vitality: Determines HP, resistance of disease.
+
+### Contested Checks
+
+Contested Checks are initiated when two opposing forces are attempting to
+accomplish something at the same time. This can be someone attempting to push
+someone else over, a Player trying to convince an NPC that they are an ally,
+or with the use of certain Powers. 
+
+For the most updated reference and ruling on Contested Checks, visit the
+[Full Player Guide](https://github.com/DeckofAdventures/TheGame/blob/main/1_Mechanics/01_PlayerGuide_Full.md#contested-checks)
+
+Here are some Examples of Skills that are used in Contested Checks, though you
+are free to decide what makes the most sense in a given scenario at your
+table. 1. Tackle Target: Attacker Finesse vs. Target Athletics 2. Convince
+Target, truthfully: Bluffing vs. Target Detection 3. Convince Target,
+deceptively: Bluffing vs. Target Detection
+
+### Epic Events 
+
+Epic Events are unique, time-gated scenarios that can occur throughout an
+Adventure. As the Game Master, you determine when an Epic Event will happen,
+set the Difficulty, and decide the Stakes. 
+
+#### How to Identify an Epic Event
+
+Epic Events occur within a specific set amount of time, typically best for
+anything that would unfold in less than a minute. They invoke a specific set
+of rules for determining the outcome, which is different than traditional
+combat, social interaction, or exploration. 
+
+Setting an Epic Event during a session typically is a good way to engage all
+of your players in a faster-paced, more free-form set of roleplaying. Since
+Players can't make a Check two times in a row, it requires active dialogue,
+planning, and participation from every Player. It is recommended to stick to
+one or two Epic Events per session, so as to keep it feeling unique and not
+drain too much Stamina from the Players, though as the Game Master you can
+set as many as you would like. 
+
+An Epic Event can be the group attempting to escape a crumbling fortress, or
+escape an island with an erupting volcano. It can cover things that are
+smaller in scope, such as a single point on a larger battlefield, or it can
+determine the outcome of an entire battle itself. As the Game Master, you
+will get a feel for what makes for a good Epic Event, and how to place it in
+your narrative to keep the session feeling fresh and lively for all of your
+Players.
+
+For the most updated reference on Epic Events, visit the [Full Player Guide](https://github.com/DeckofAdventures/TheGame/blob/main/1_Mechanics/01_PlayerGuide_Full.md#Epic-Events)
```

Comment:
I don't want to imply that a doc we write is out of date. The metadata can handle that.

```suggestion
For more information on Epic Events, visit the [Full Player Guide](https://github.com/DeckofAdventures/TheGame/blob/main/1_Mechanics/01_PlayerGuide_Full.md#Epic-Events)
```